src/components/footer/config/index.ts

```diff
@@ -0,0 +1,656 @@
+            fr: 'https://swissmarketplace.group/fr/automotive/',
+            it: 'https://swissmarketplace.group/it/automotive/',
+          },
+          target: '_blank',
+        },
+        {
+          translationKey: 'footer.sections.aboutUs.jobs',
+          visibilitySettings: {
+            brand: { as24: true, ms24: true },
+          },
+          link: {
+            de: 'https://swissmarketplace.group/de/karriere/',
+            en: 'https://swissmarketplace.group/career/',
+            fr: 'https://swissmarketplace.group/fr/carrieres/',
+            it: 'https://swissmarketplace.group/career/',
+          },
+          target: '_blank',
+        },
+      ],
+    },
+    {
+      title: [
+        {
+          translationKey: 'footer.sections.advertisement.title',
+          visibilitySettings: {
+            brand: { as24: true, ms24: true },
+          },
+        },
+      ],
+      items: [
+        {
+          translationKey: 'footer.sections.advertisement.smg',
+          visibilitySettings: {
+            brand: { as24: true, ms24: true },
+          },
+          link: {
+            de: 'https://swissmarketplace.group/de/werbung/',
+            en: 'https://swissmarketplace.group/advertising/',
+            fr: 'https://swissmarketplace.group/fr/publicite/',
+            it: 'https://swissmarketplace.group/advertising/',
+          },
+          target: '_blank',
+        },
+      ],
+    },
+    {
+      title: [
+        {
+          translationKey: 'footer.sections.media.title',
+          visibilitySettings: {
+            brand: { as24: true, ms24: true },
+          },
+        },
+      ],
+      items: [
+        {
+          translationKey: 'footer.sections.media.contact',
+          visibilitySettings: {
+            brand: { as24: true, ms24: true },
+          },
+          link: {
+            de: 'https://medien-de.autoscout24.ch/',
+            en: 'https://medien-de.autoscout24.ch/',
+            fr: 'https://medien-fr.autoscout24.ch/',
+            it: 'https://medien-de.autoscout24.ch/',
+          },
+          target: '_blank',
+        },
+      ],
+    },
+    {
+      title: [
+        {
+          translationKey: 'footer.sections.legal.title',
+          visibilitySettings: {
+            brand: { as24: true, ms24: true },
+          },
+        },
+      ],
+      items: [
+        {
+          translationKey: 'footer.sections.legal.securityHint',
+          visibilitySettings: {
+            brand: { as24: true, ms24: true },
+          },
+          link: {
+            de: '/de/c/d/information/sicherheitshinweise-fuer-kaeufer-und-verkaeufer-so-tappen-sie-nicht-in-die-betrugsfalle?a=13444',
+            en: '/de/c/d/information/sicherheitshinweise-fuer-kaeufer-und-verkaeufer-so-tappen-sie-nicht-in-die-betrugsfalle?a=13444',
+            fr: '/fr/c/d/information/conseils-de-s%C3%A9curit%C3%A9-d-autoscout24-prot%C3%A8gez-vous-contre-la-fraude?a=11818',
```

Comment:
This one does not work, it's because there are accents (`é` and `è`) in the URL: https://www.autoscout24.ch/fr/c/d/information/conseils-de-sécurité-d-autoscout24-protègez-vous-contre-la-fraude?a=11818
But well, it does not work for AS24 in Prod currently 🙈 

For MS24, it works in Prod with this link: https://www.motoscout24.ch/fr/c/d/information/conseils-de-sècuritè-d-autoscout24-protègez-vous-contre-la-fraude?a=13948 (accents are completely off though)

If I change `motoscout` for `autoscout` in the working URL, there is no more error, we are on AS24 according to the header but the content of the page is MotoScout.

Reply:
Thanks, I asked Marlise to provide the right link